state-sync/src/executor_proxy.rs

```diff
@@ -182,7 +179,7 @@ impl ExecutorProxyTrait for ExecutorProxy {
             .get_transactions(starting_version, limit, target_version, false)
     }
 
-    fn get_epoch_proof(&self, epoch: u64) -> Result<LedgerInfoWithSignatures> {
+    fn get_epoch_change_ledger_info(&self, epoch: u64) -> Result<LedgerInfoWithSignatures> {
```

Comment:
Isn't `Proof` `LedgerInfo`?  I don't think it's quite clear to the reader, but you need to ask someone like Zekun

Reply:
So, internally the method gets the epoch change proof from storage, but then strips off the last ledger info from the proof. We could go with either option, but I agree.. in general this isn't super. I'd need to understand the very fine details before I could suggest refactoring the structure 🤔

Reply:
either seems fine to me